### PR TITLE
AADServicePrincipal

### PR DESCRIPTION
**Pull request by jdlagace**

### CODE REVIEW

**Comment by FabienTschanz** [comment]
@jdlagace Please update your PR with the latest changes and fix the merge conflicts in the changelog. Thank you.

File: CHANGELOG.md [hunk 1]
```diff
@@ -69,8 +69,17 @@
 * AADLifecycleWorkflowSettings
   * Initial release.
 * AADServicePrincipal
+<<<<<<< HEAD
```
Comment (ykuijs) [inline CHANGELOG.md]: The Merge Conflict hasn't been resolved correctly. Please replace the following lines:
```Powershell
<<<<<<< HEAD
<<<<<<< HEAD
  * Added support for SAML.
  * Fix issue with Owners.
=======
  * Adding Delegated Permission Classification Property
>>>>>>> 120553154f48aa5f7a810d7e4cc1583c3fdbb067
=======
  * Added Delegated Permission Classification Property
  * Added Custom Security Attributes Property
>>>>>>> cbe5a16c113582a9a35ca3034c5baa4602ee393e
```
with:
```Powershell
  * Added support for SAML.
  * Fix issue with Owners.
  * Added Delegated Permission Classification Property
  * Added Custom Security Attributes Property
```

**Review by ykuijs — CHANGES_REQUESTED** [review]
Please correct merge conflict

File: CHANGELOG.md (hunk 1 above)
Comment (jdlagace) [reply CHANGELOG.md]: @ykuijs Should be fixed now, thanks!

**Comment by jdlagace** [comment]
Will create a new PR with fixes

**Pull request closed without merging**